cli/crates/cli/tests/utils/environment.rs

```diff
@@ -104,7 +104,10 @@ impl Environment {
     pub fn set_variables(&mut self, variables: HashMap<String, String>) {
         let env_file = variables
             .into_iter()
-            .fold(String::new(), |_, (key, value)| format!(r#"{key}="{value}""#));
```

Comment:
This had a bug where only one var would be written to the file.

Reply:
Seems like it was missing appending the accumulator in the fold